<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LFilterForm</class>
 <widget class="QDialog" name="LFilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set kernel</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Kernel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Added value&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="addedVBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="kernelWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>70</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>1</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>1</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>addedVBox</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
